plugins/smqtk/smqtk_process_query.py

```diff
@@ -147,6 +151,41 @@ def add_config_traits(self):
         )
         self.declare_config_using_trait('query_return_size')
 
+    def get_svm_bytes(self):
+        svm_model = self.iqr_session.rel_index.get_model()
+        tmp_file_name = "tmp_svm.model"
+
+        svmutil.svm_save_model(tmp_file_name, svm_model)
```

Comment:
Can this be done without a temporary file using `io.BytesIO`? If not might it be better to use the `tempfile` module?